app/reducers/review.jsx

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios'
+import initialState from '../initialState'
+
+const reducer = (state = initialState, action) => {
+  const newState = Object.assign({}, state)
+
+  switch (action.type) {
+  case SELECT_REVIEWS:
+    newState.reviews = action.reviews
+    break
+  case ADD_REVIEW:
+    newState.reviews = [...newState.reviews, action.review]
```

Comment:
@zcaceres changed this case statement to use .concat() instead of ES6 destructuring